notebooks/README.md

```diff
@@ -6,3 +6,4 @@ Here are some notebooks compiled for users to better leverage the library capabi
 |:----------|:-------------|------:|
 | [Quicktour](https://github.com/mindee/notebooks/blob/main/doctr/quicktour.ipynb) | A presentation of the main features of docTR | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/mindee/notebooks/blob/main/doctr/quicktour.ipynb) |
 | [Export as PDF/A](https://github.com/mindee/notebooks/blob/main/doctr/export_as_pdfa.ipynb) | Produce searchable PDFs from docTR results | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/mindee/notebooks/blob/main/doctr/export_as_pdfa.ipynb) |
+[Artefact Plot Predicitons](https://github.com/mindee/notebooks/blob/main/doctr/artefact_detection.ipynb) | Object detection for artefacts in documents | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/mindee/notebooks/blob/main/doctr/artefact_detection.ipynb) |
```

Comment:
Careful, there is a typo "Artefact Plot Predicitons" --> "Artefact detection"

Reply:
Thanks :)